lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,12 @@ def initialize(engine_config:, directory:)
 
         def run
           files.map do |file|
-            process_file(file)
+            begin
+              process_file(file)
+            rescue => e
```

Comment:
The reason I didn't do it in `process_file` is because each language defines that method themselves.

I don't think it's worth duplicating this in each analyzer. Thoughts?